frontend/src/views/web/landing.html

```diff
@@ -201,10 +223,10 @@ <h4 class="w-300 center">Partner Organizations</h4>
                                 </div>
                             </div>
                         </div>
-                        </div>
                     </div>
                 </div>
             </div>
+        </div>
```

Comment:
```suggestion
```

Would suggest you remove this tag. This is not something you have introduced, but ideally good to remove this.
I just realised when web storm showed this error of mismatching div.
<img width="343" alt="Screenshot 2019-04-18 at 6 45 54 PM" src="https://user-images.githubusercontent.com/11664259/56395987-dd175680-620a-11e9-87ba-1c8b5ff5ca78.png">